openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx

```diff
@@ -108,8 +110,8 @@ const RuleForm: FC<RuleFormProps> = ({ ruleData, setRuleData }) => {
     return option;
   }, [ruleData.resources, policyResources]);
 
-  const getConditionOptions = (funtions: Function[]) => {
-    return funtions.reduce((prev: BaseOptionType[], curr: Function) => {
+  const getConditionOptions = (functions: Function[]) => {
```

Comment:
what is functions?

Reply:
functions are condition functions like `isOwner()`.